features/support/env.rb

```diff
@@ -21,9 +21,19 @@
 Capybara.register_driver :selenium do |app|
   require 'selenium/webdriver'
   Selenium::WebDriver::Firefox::Binary.path = ENV['FIREFOX_BINARY_PATH'] || Selenium::WebDriver::Firefox::Binary.path
-  Capybara::Selenium::Driver.new(app, :browser => :firefox)
+  Capybara::Selenium::Driver.new(app, browser: :firefox)
 end
 
+Capybara.register_driver :mobile do |app|
+  require 'selenium/webdriver'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.